components/breadcrumb/index.jsx

```diff
@@ -33,18 +35,32 @@ const defaultProps = {
 const Breadcrumb = (props) => {
 	checkProps(BREADCRUMB, props, componentDoc);
 
-	const { trail } = props;
+	const { parentIndex } = props;
+	let { trail } = props;
 	const assistiveText =
 		typeof props.assistiveText === 'string'
 			? props.assistiveText
 			: {
 					...defaultProps.assistiveText,
 					...props.assistiveText,
 				}.label;
-
+	let menuTrail = [];
+	if (parentIndex) {
+		menuTrail = trail.slice(0, parentIndex);
+		menuTrail = menuTrail.map((item) => ({
+			label: item.props.children,
+			value: item.props.children,
+		}));
+		trail = trail.slice(parentIndex);
+	}
 	return (
 		<nav role="navigation" aria-label={assistiveText}>
 			<ol className="slds-breadcrumb slds-list_horizontal">
+				{menuTrail.length > 0 && (
```

Comment:
Could we instead expose a `BreadcrumbDropdown` instead of `parentIndex` that the consumer populates for the overflow menu? That way, it would simplify the code a lot (no need of splitting `trail`) and users won't need to keep track of an index where the cut should happen for overflow items? What do you think?

Reply:
That can be done too. I will update the code accordingly and expose the `BreadcrumbDropdown` subcomponent.